### PR TITLE
Added github stream to the main page

### LINKED ISSUE #22: As an author, I want to pull in my github activity to my “stream”

### PR DESCRIPTION
**Pull request by ColeBoytinck**
Closes #22

### CODE REVIEW

File: web_server/social_distribution/utils/endpoint_utils.py [hunk 1]
```diff
@@ -169,11 +169,11 @@ def handle(self, request):
...
-        if self.requires_authentication and not request.user.is_authenticated:
-            return JsonResponse({
-                "success": False,
-                "message": "You must be logged in to access this Endpoint"
-            }, status=403)
+        #if self.requires_authentication and not request.user.is_authenticated:
+         #   return JsonResponse({
+          #      "success": False,
+           #     "message": "You must be logged in to access this Endpoint"
+            #}, status=403)
```
Comment (AustinGrey) [inline web_server/social_distribution/utils/endpoint_utils.py]: Why are you disabling login checks?

File: web_server/friendship/migrations/0001_initial.py [hunk 2]
```diff
@@ -11,15 +11,37 @@ class Migration(migrations.Migration):
...
+            },
+        ),
         migrations.CreateModel(
             name='Friend',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('author_id', models.URLField(max_length=500)),
-                ('friend_id', models.URLField(max_length=500)),
+                ('author_id', models.CharField(max_length=500)),
+                ('friend_id', models.CharField(max_length=500)),
```
Comment (AustinGrey) [inline web_server/friendship/migrations/0001_initial.py:30]: Something is off here, why are you changing the friendship models to implement the github feature? Did you accidentally make migrations on old model files?

**Review by AustinGrey — CHANGES_REQUESTED** [review]
There are lots of un-related changes changes to the github feature, with no explanation. Were some files accidentally committed?

File: web_server/social_distribution/utils/endpoint_utils.py (hunk 1 above)
Comment (ColeBoytinck) [reply web_server/social_distribution/utils/endpoint_utils.py]: I was testing and I forgot to remove the comments

File: web_server/friendship/migrations/0001_initial.py (hunk 2 above)
Comment (ColeBoytinck) [reply web_server/friendship/migrations/0001_initial.py:30]: I am not actually sure what is up with the migrations. I had deleted everything and re migrated, as I was having DB conflicts before. Maybe master has some weird migration?

**Pull request closed without merging**